src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -61,6 +64,32 @@ export default defineComponent({
         items: {
             type: Array as PropType<DlLabelPickerItem[]>,
             default: () => [] as PropType<DlLabelPickerItem[]>
+        },
+        /**
+         * Props for the empty state component
+         */
+        emptyStateProps: {
+            type: Object as PropType<DlEmptyStateProps>,
+            default: () =>
+                ({
+                    title: '',
+                    subtitle: 'No data to show yet.',
+                    icon: 'icon-dl-dataset-filled'
+                } as unknown as PropType<DlEmptyStateProps>)
+        },
+        /**
+         * Don't show the footer
```

Comment:
Add better descriptive jsdoc for each prop. 

Example:
`Hides the bottom footer of the DlLabelPicker`